addons/account_check_printing/models/account_journal.py

```diff
@@ -32,6 +32,10 @@ def _default_outbound_payment_methods(self):
         compute='_compute_check_next_number',
         inverse='_inverse_check_next_number',
         help="Sequence number of the next printed check.",
+        pattern=(
+            _('Next Check Number should only contains numbers.'),
```

Comment:
Is this a new field attribute? As for `string` and `help`, translation support should be done by the ORM

Reply:
Victor is right, you should not call the `_()` function when defining a class, because there is no context to translate to.  And this looks like a new type of translation, by the way.

Reply:
> And this looks like a new type of translation, by the way.

We could store it on `ir.model.fields` to avoid this (as we already do for string and help).

Reply:
Better like this?